src/lib/service/allowedAttestationTitles.ts

```diff
@@ -0,0 +1,61 @@
+import { encodeFunctionData } from "viem";
+
+import { TRUSTFUL_SMART_CONTRACT_ADDRESS } from "../client/constants";
+import { publicClient } from "../wallet/wallet-config";
+
+export interface ConnetedWalletConfiguration {
+  walletClient: any;
+  chain: number;
+}
+
+export async function allowedAttestationTitles(
+  attestationTitle: string,
+  configurations: ConnetedWalletConfiguration,
+  msgValue: bigint,
+) {
+  const data = encodeFunctionData({
+    abi: [
+      {
+        type: "function",
+        name: "allowedAttestationTitles",
+        inputs: [{ name: "title", type: "string", internalType: "string" }],
+        outputs: [{ name: "", type: "bool", internalType: "bool" }],
+        stateMutability: "view",
```

Comment:
This function `allowedAttestationTitles ` is a view type, so we won't return a receipt but rather the read value from the chain. We can't also estimate the gas of view functions. So remove the gas estimation as gas parameters, and change the return type to be like a view type. You can compare to how the file `getSwap.ts` was built on Swaplace for reference